private/buf/cmd/buf/command/beta/price/price.go

```diff
@@ -144,46 +118,27 @@ func run(
 	if err != nil {
 		return err
 	}
-	statsSlice := make([]*protostat.Stats, len(moduleConfigs))
+	request := &registryv1alpha1.GetPriceEstimationTextRequest{
+		Modules: make([]*modulev1alpha1.Module, len(moduleConfigs)),
+	}
 	for i, moduleConfig := range moduleConfigs {
-		stats, err := protostat.GetStats(ctx, bufmodulestat.NewFileWalker(moduleConfig.Module()))
+		protoModule, err := bufmodule.ModuleToProtoModule(ctx, moduleConfig.Module())
 		if err != nil {
 			return err
 		}
-		statsSlice[i] = stats
+		request.Modules[i] = protoModule
 	}
-	tmpl, err := template.New("tmpl").Parse(tmplCopy)
+	service := connectclient.Make(
+		clientConfig,
+		// TODO: there must be some constant here somewhere
+		// We only want to hit the public one for this function
+		"buf.build",
```

Comment:
```suggestion
		bufconnect.DefaultRemote,
```

Reply:
Commandeer this PR, no need for suggestions lol :-)

Reply:
sure